scenario/tasks.yaml

```diff
@@ -10,7 +10,9 @@ tasks:
    act_stop_vms: ['StopVms', 'src_cloud']
    act_start_vms: ['StartVms', 'dst_cloud']
    act_start_vms_if_needed: ['StartVmsIfNeeded', 'dst_cloud']
-   act_dissociate_floatingip: ['DissociateFloatingip', 'src_cloud']
+   act_dissociate_floatingip: ['DisassociateAllFloatingips', 'src_cloud']
+   disassociate_floatingip_on_source: ['DisassociateFloatingip', 'src_cloud']
```

Comment:
I decided not to change the naming of the original task, while for the new one it seems more readable to have it named `disassociate_floatingip_on_source`